Makefile

```diff
@@ -38,9 +38,7 @@ decrement_version:
 	$(PYTHON) util/create_version_file.py --decrement
 
 init: ## pulls submodules and initializes virtual environment
-	git submodule update --init --recursive
-	which pdm || pip install --user pdm
-	pdm venv create -n $(PROJECT)
```

Comment:
Does this need to be changed? The new version no longer attempts to install pdm

Reply:
`pip install --user pdm` is not in compliance with `PEP 668`, and this only seems to be an issue with newer OS versions (e.g., Ubuntu 24.04). I have a workaround on my own machine, but different users may prefer to install `pdm` differently.